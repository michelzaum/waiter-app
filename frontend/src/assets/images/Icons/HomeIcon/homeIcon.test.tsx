import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { HomeIcon } from '.';

describe('HomeIcon', () => {
  it('Should render the HomeIcon', () => {
    //given
    const result = render(<HomeIcon />);

    // then
    expect(result).toMatchSnapshot();
  });
});
