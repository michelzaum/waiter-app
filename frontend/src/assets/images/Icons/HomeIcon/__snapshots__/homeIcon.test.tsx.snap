// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeIcon > Should render the HomeIcon 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        fill="none"
        height="24"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M19.5282 7.98627L13.7682 3.50595C12.7285 2.69667 11.2722 2.69667 10.2315 3.50595L4.47154 7.98627C3.76978 8.53251 3.35986 9.37155 3.35986 10.2595V17.2992C3.35986 18.8899 4.64914 20.1792 6.23986 20.1792H17.7599C19.3506 20.1792 20.6399 18.8899 20.6399 17.2992V10.2595C20.6399 9.37155 20.2299 8.53251 19.5282 7.98627Z"
          fill-rule="evenodd"
          stroke="currentColor"
          stroke-width="1.5"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M19.5282 7.98627L13.7682 3.50595C12.7285 2.69667 11.2722 2.69667 10.2315 3.50595L4.47154 7.98627C3.76978 8.53251 3.35986 9.37155 3.35986 10.2595V17.2992C3.35986 18.8899 4.64914 20.1792 6.23986 20.1792H17.7599C19.3506 20.1792 20.6399 18.8899 20.6399 17.2992V10.2595C20.6399 9.37155 20.2299 8.53251 19.5282 7.98627Z"
        fill-rule="evenodd"
        stroke="currentColor"
        stroke-width="1.5"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
